CHANGELOG.md

```diff
@@ -34,6 +34,7 @@
   * `cortex_alertmanager_sync_configs_failed_total`
   * `cortex_alertmanager_tenants_discovered`
   * `cortex_alertmanager_tenants_owned`
+* [ENHANCEMENT] Return server side performance metrics for query-frontend (using Server-timing header). #3685
```

Comment:
We've just released `1.7.0-rc.0`. Could you rebase `master` and move the CHANGELOG entry under the `master / unreleased` section, please?

Reply:
Done! :D